src/repart/repart.c

```diff
@@ -4326,7 +4326,7 @@ static int prepare_temporary_file(Context *context, PartitionTarget *t, uint64_t
                 return log_error_errno(fd, "Failed to create temporary file: %m");
 
         r = read_attr_fd(fdisk_get_devfd(context->fdisk_context), &attrs);
-        if (r < 0 && !ERRNO_IS_NEG_NOT_SUPPORTED(r))
+        if (r < 0 && !ERRNO_IS_NEG_NOT_SUPPORTED(r) && r != -EOVERFLOW)
```

Comment:
I'd prefer to log `EOVERFLOW` and ignore it. But since we silently ignore `NOT_SUPPORTED` family it's probably fine.

Reply:
i think we should not check for EOVERFLOW explicitly. just turn this error into a warning, and don't exit the function here? don't forget to add `, ignoring` to the log message.

i.e. this is really brokeness of docker which might be ok to handle gracefully, but certainly still with logging